test/model/binding.dart

```diff
@@ -513,34 +512,6 @@ class FakeFlutterLocalNotificationsPlugin extends Fake implements FlutterLocalNo
     return true;
   }
 
-  FlutterLocalNotificationsPlatform? _platform;
-
-  @override
-  T? resolvePlatformSpecificImplementation<T extends FlutterLocalNotificationsPlatform>() {
```

Comment:
nit in commit message:

    notif [nfc]: Remove unused test harness for flutter_local_notification

This sounds like it might mean our whole test harness for that package. Also the package's name has an "s" at the end. So:

    notif [nfc]: Remove unused test harness for parts of flutter_local_notifications